Terminal.Gui/Drawing/ColorScheme.cs

```diff
@@ -52,19 +52,39 @@ public ColorScheme (Attribute attribute)
     private readonly Attribute _normal;
 
     /// <summary>The default foreground and background color for text when the view is disabled.</summary>
-    public Attribute Disabled { get => _disabled; init => _disabled = value; }
+    public Attribute Disabled
+    {
+        get => _disabled;
+        init => _disabled = value;
+    }
 
     /// <summary>The foreground and background color for text when the view has the focus.</summary>
-    public Attribute Focus { get => _focus; init => _focus = value; }
+    public Attribute Focus
+    {
+        get => _focus;
+        init => _focus = value;
+    }
 
     /// <summary>The foreground and background color for for text in a focused view that indicates a <see cref="View.HotKey"/>.</summary>
-    public Attribute HotFocus { get => _hotFocus; init => _hotFocus = value; }
+    public Attribute HotFocus
```

Comment:
ReSharper is capable of turning these into auto-properties, with appropriate settings.